### ISSUE TITLE
feature request: read partitioned parquet file as a single LazyFrame

### ISSUE THREAD

**Issue by Sage0614** [danielgafni/dagster-polars#58]
Hi,

As far as I am aware, for a partioned LazyFrame/DataFrame, you can only read it as DataFramePartitions|LazyFramePartitions, is it possible to read partioned parquet as a single LazyFrame?

polars 0.20.5 support syntax like wildcard read for all parquet file under a folder like:
```python
pl.scan_parquet("folder/*parquet")
```
so on polars side, user's direct reading each partitioned parquets as a dictionary of files is not necessary now.

**Comment by danielgafni** [comment]
Hey! 

I would have to submit a workaround for this to work.
Currently the `UPathIOManager` doesn't allow this (I opened an issue for this [here](https://github.com/dagster-io/dagster/issues/19220)). 
I did a workaround with `PolarsDeltaIOManager` to support loading multiple partitions as a single `LazyFrame`, but haven't done it for the `PolarsParquetIOManager` yet. 
I think this could be done in a few days.
